packages/@storefront/core/test/unit/services/autocomplete.ts

```diff
@@ -173,6 +178,19 @@ suite('Autocomplete Service', ({ expect, spy, stub, itShouldBeCore, itShouldExte
 
       expect(service.hasActiveSuggestion()).to.be.false;
     });
+
+    it('should return false if no autocomplete tags has an active selection, but hoverAutoFill is off', () => {
```

Comment:
Should this test read "...if any autocomplete tags..."? If I understand correctly, we're asserting that `hasActiveSuggestion()` will return false in the case where 1 or more tags are active, but `hoverAutoFill` is disable?